app/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "static-app-template",
+  "private": true,
+  "description": "A template for creating static apps with Gatsby.",
+  "version": "0.1.0",
+  "author": "DataMade",
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.17",
+    "@fortawesome/free-regular-svg-icons": "^5.8.1",
+    "@fortawesome/free-solid-svg-icons": "^5.8.1",
+    "@fortawesome/react-fontawesome": "^0.1.4",
+    "bootstrap": "^4.3.1",
```

Comment:
I'm not totally sold on the React Bootstrap/Fontawesome plugins, but I've included them here to acknowledge that these are important components of our stack. (Using this package configuration we can also use Bootstrap/FA directly without having to use the React components, too. Since Gatsby supports [code splitting](https://reactjs.org/docs/code-splitting.html), these dependencies won't be built into the final bundle unless pieces of the code import them directly.)